src/alchemlyb/parsing/parquet.py

```diff
@@ -1,9 +1,42 @@
 import pandas as pd
+from loguru import logger
 
 from . import _init_attrs
 
 
-@_init_attrs
```

Comment:
Is the decorator not needed anymore for `extract_u_nk()`?

Reply:
Not anymore as the attrs are loaded from the parquet file

Reply:
thanks for the explanation